### DISCUSSION TITLE
AmberELEC - 20230131 Panda Conspiracy

### DISCUSSION THREAD

**Discussion by dhwz** [351ELEC/351ELEC#946] [General]
# Release Notes
### What's new?
* Newly added Libretro cores (all devices):
	* **flycast 2021** (Sega Dreamcast / Naomi, Atomiswave)
	* **MAME 2016** (0.174 rom set)
	* **beetle_pce** (NEC Engine / PC Engine CD / TurboGrafx 16 / TurboGrafx CD / SuperGrafx)
	* **DOSBox Core** (MS-DOS)
	* **duckstation** (Sony Playstation)
	* **fake-08** (Pico-8)
	* **ep128emu** (Amstrad CPC, ZX Spectrum, Enterprise 64/128, TVC)
	* **emuscv** (Super Cassette Vision)
	* **jaxe** (CHIP-8)
	* **wasm-4** (WASM-4)
	* **lowresnx** (LowRes NX)
	* **mojozork** (Z-Machine)
* Newly added Libretro cores (RG552 only):
	* **bsnes** (Nintento Game Boy / Game Boy Color / MSU-1 / SNES / SFC / Satellaview)
	* **beetle-snes** (Nintento  SNES / SFC / Satellaview)
 	* **bsnes2014 performance** (Nintento SNES / SFC / Satellaview)
 	* **bsnes2014 balanced** (Nintento SNES / SFC / Satellaview)
 	* **bsnes mercury performance** (Nintento SNES / SFC / Satellaview)
 	* **bsnes mercury  balanced** (Nintento SNES / SFC / Satellaview)
	* **mesen** (Nintento NES / Famicom)
	* **mesen-s** (Nintento Game Boy / Game Boy Color / MSU-1 / SNES / SFC / Satellaview)
* Added new options and some new supported cores for  **RetroRun**
* Added an utility to set the time offline
* Added an utility to migrate **Libretro PPSSPP core** save files to its new location
* Added RetroAchievements support for **Arduboy**, **Dreamcast**, **Channel F**, **Coleco Vision**, **Mega Duck**

### What has changed?
* Updated **RetroArch** to v1.14.0
* Updated **SDL2** to v2.26.2
* Updated most **Libretro cores** and **standalone emulators** to their latest version
* Updated **MAME - Current core** (0.251 rom set)
* Updated **RetroRun** for better system performance
* Removed **DOSBox SVN core** in favor of **DOSBox Core**
* Some improvements were done to **RetroArch**'s GUI for RG552 (Battery support + proper UI scaling)
* Set default deadzone for **Raze**, **ECWolf**, **Mupen64plus standalone**
* Added m3u support for mplayer
* Updated community shaders
* **Libretro PPSSPP core** save files are now stored in the **PPSSPP standalone** folder `/roms/gamedata/ppsspp`
* General system stability and performance improvements to enhance the user's experience

### Fixes:
* Fixed SDL2 cursor issue on 640x480+ devices
* Fixed anti-piracy detection on some PSP games
* Fixed issue on the 351P where the audio was too low
* Fixed issue with volume control not working
* Fixed skip/back on mplayer with the 351P

### Things to do after an upgrade:
* If you've used the **Libretro PPSSPP core** before, move your save files with the **PPSSPP migration tool** to its new location.

**IMPORTANT NOTE**: There are **four different images** below, one for the **RG351P/M**, **RG351V**, **RG351MP** and **RG552**! 

**If you download the incorrect image for your device, it will not boot!**  If you are unsure, use the the following links:
**New Installations** (`.img.gz`):  **[RG351P/M](https://github.com/AmberELEC/AmberELEC/releases/download/20230131/AmberELEC-RG351P.aarch64-20230131.img.gz)** |  **[RG351V](https://github.com/AmberELEC/AmberELEC/releases/download/20230131/AmberELEC-RG351V.aarch64-20230131.img.gz)** |  **[RG351MP](https://github.com/AmberELEC/AmberELEC/releases/download/20230131/AmberELEC-RG351MP.aarch64-20230131.img.gz)** | **[RG552](https://github.com/AmberELEC/AmberELEC/releases/download/20230131/AmberELEC-RG552.aarch64-20230131.img.gz)**
**Upgrades** (place in `/storage/roms/update`): **[RG351P/M](https://github.com/AmberELEC/AmberELEC/releases/download/20230131/AmberELEC-RG351P.aarch64-20230131.tar)** |  **[RG351V](https://github.com/AmberELEC/AmberELEC/releases/download/20230131/AmberELEC-RG351V.aarch64-20230131.tar)** |  **[RG351MP](https://github.com/AmberELEC/AmberELEC/releases/download/20230131/AmberELEC-RG351MP.aarch64-20230131.tar)** | **[RG552](https://github.com/AmberELEC/AmberELEC/releases/download/20230131/AmberELEC-RG552.aarch64-20230131.tar)**

<hr /><em>This discussion was created from the release <a href='https://github.com/AmberELEC/AmberELEC/releases/tag/20230131'>AmberELEC - 20230131 Panda Conspiracy</a>.</em>

**Comment by stonegrimm** [comment]
Awesome, thanks a bunch!

**Comment by dtammam** [comment]
Thank you 😊

**Comment by GorillaWarHead** [comment]
Awesome 🎉
